File: arrays/array_update_elements/array_update_elements.java
package array_update_elements;

public class array_update_elements {
    
    public static void array_update(int A[],int k,int item){
        A[k] = item;
        int n = A.length;
        for(int i=0;i<n;i++){
            System.out.printf("%d th element of A is %d\n",i,A[i]);
        }
    }

    public static void main(String args[]){
        int A[] = {1,2,3,4,5,6,7};
        int n = A.length;
        int k = 4;
        int item = 10;
        System.out.println("Before update:\n");
        for(int i=0;i<n;i++){
            System.out.printf("%d th element of A is %d\n",i,A[i]);
        }
        System.out.println("After update:\n");
        array_update(A,k,item);


    }
}

File: arrays/array_update_elements/array_update_elements.c
/**
 * this code updates the element at a particular
 * index in an array in C language
 * 
 */

#include<stdio.h>
void display_array(int A[],int n);
void array_update(int A[],int n,int k,int item){
    //update the element
    A[k] = item;

    //print the array
    for(int i=0;i<n;i++){
        printf("%dth element of A is %d\n",i,A[i]);
    }
}

void display_array(int A[],int n){
    for(int i=0;i<n;i++){
        printf("%dth element of A is %d\n",i,A[i]);
    }
}

int main(){
    int A[]  = {1,2,3,4,5,6,7};
    int n = 7;
    int k = 5;
    int item = 10;
    printf("Before update\n");
    display_array(A,n);
    printf("After update\n");
    array_update(A,n,k,item);

    }
